Source/Core/DolphinQt/Settings/WiiPane.cpp

```diff
@@ -221,6 +221,8 @@ void WiiPane::LoadConfig()
 
 void WiiPane::OnSaveConfig()
 {
+  Config::ConfigChangeCallbackGuard guard;
```

Comment:
Why is this one called `guard` but the others are `config_guard`? :P

Reply:
I don't know. :P